clarity.d.ts

```diff
@@ -266,6 +266,9 @@ interface IInstrumentationEventState {
 interface IJsErrorEventState extends IInstrumentationEventState {
   source: string;
   message: string;
+  stack: string;
```

Comment:
sure, I think that Sarvesh is also moving the version number up - let me wait for his to merge then I will change mine.

---
In reply to: [119245274](https://github.com/Microsoft/clarity-js/pull/9#discussion_r119245274) [](ancestors = 119245274)